lib/modules/manager/mix/__fixtures__/mix.exs

```diff
@@ -20,18 +20,18 @@ defmodule MyProject.MixProject do
 #   {:broadway_dashboard, "~> 0.2.2"},
       #    {:broadway_dashboard, "~> 0.2.2"},
       {:postgrex, "~> 0.8.1"}, #  {:broadway_dashboard, "~> 0.2.2"},
-      {:ecto, ">2.1.0 or <=3.0.0"},
+      {:foo_bar, ">2.1.0 or <=3.0.0"},
```

Comment:
why rename deps?

Reply:
This fixture is an invalid `mix.exs` file.

Here's a cleaned version of the original (removed fake private packages and updated the tag for `cowboy` to a real one). The screenshot below shows the output when trying to run  `mix deps.get` - it doesn't work because of duplicated dependencies. I thought it would be important to have the fixture reflect a real life example.

```elixir
defmodule MyProject.MixProject do
  use Mix.Project

  def project do
    [
      app: :my_project,
      version: "0.1.0",
      elixir: "~> 1.14",
      start_permanent: Mix.env() == :prod,
      deps: deps()
    ]
  end

  # Run "mix help compile.app" to learn about applications.
  def application do
    [
      extra_applications: [:logger]
    ]
  end

  # Run "mix help deps" to learn about dependencies.
  defp deps() do
    [ #{:broadway_dashboard, "~> 0.2.2"},
#{:broadway_dashboard, "~> 0.2.2"},
#   {:broadway_dashboard, "~> 0.2.2"},
      #    {:broadway_dashboard, "~> 0.2.2"},
      {:postgrex, "~> 0.8.1"}, #  {:broadway_dashboard, "~> 0.2.2"},
      {:ecto, ">2.1.0 or <=3.0.0"},
      {:cowboy, github: "ninenines/cowboy", tag: "2.11.0"},
      {:phoenix, git: "https://github.com/phoenixframework/phoenix.git", branch: "main"},
      {:ecto, github: "elixir-ecto/ecto", ref: "795036d997c7503b21fb64d6bf1a89b83c44f2b5"},
      # {:secret, "~> 1.0", organization: "acme"},
      # {:also_secret, "~> 1.0", only: [:dev, :test], organization: "acme", runtime: false},
      {:ex_doc, ">2.1.0 and <=3.0.0"},
      {:jason, ">= 1.0.0"},
      {:jason, "~> 1.0",
        optional: true},
      {:hammer_backend_redis, "~> 6.1"},
      {:phoenix, "== 1.6.14"}
    ]
  end
end
```


![image](https://github.com/renovatebot/renovate/assets/105243742/e6355575-82b3-4a2d-bfe5-e9f7adb32093)